widget/embedded/src/pages/Routes.tsx

```diff
@@ -0,0 +1,94 @@
+import type { SelectedQuote } from '../store/quote';
+
+import { i18n } from '@lingui/core';
+import { Divider } from '@rango-dev/ui';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { HomeButtons } from '../components/HeaderButtons';
+import { Layout, PageContainer } from '../components/Layout';
+import { QuoteSkeleton } from '../components/QuoteSkeleton';
+import { navigationRoutes } from '../constants/navigationRoutes';
+import { QuoteInfo } from '../containers/QuoteInfo';
+import { useNavigateBack } from '../hooks/useNavigateBack';
+import { useSwapInput } from '../hooks/useSwapInput';
+import { useQuoteStore } from '../store/quote';
+import { sortTags } from '../utils/quote';
+
+const ITEM_SKELETON_COUNT = 6;
+export function RoutesPage() {
+  const navigate = useNavigate();
+  const navigateBack = useNavigateBack();
+
+  const {
+    selectedQuote,
+    quotes,
+    refetchQuote,
+    setSelectedQuote,
+    updateSomeQuoteState,
+  } = useQuoteStore();
+  const {
+    fetch: fetchQuote,
+    loading: fetchingQuote,
+    error: quoteError,
+  } = useSwapInput({ refetchQuote });
+
+  const onClickOnQuote = (quote: SelectedQuote) => {
+    setSelectedQuote(quote);
+    updateSomeQuoteState('refetchQuote', false);
+
+    if (quote.tags) {
+      const sortedQuoteTags = sortTags(quote.tags);
+      updateSomeQuoteState('selectedTag', sortedQuoteTags[0]);
+    }
+    navigateBack();
+  };
+
+  return (
+    <Layout
+      hasLogo
+      header={{
+        onWallet: () => {
+          navigate(`/${navigationRoutes.wallets}`);
```

Comment:
I believe navigate(navigationRoutes.wallets) is sufficient.

Reply:
fixed